### ISSUE TITLE
  @notifyCss make changes on my styles

### ISSUE THREAD

**Issue by donchoborisov** [mckenziearts/laravel-notify#38]
When i add   @notifyCss to my head its make changes to my styles actually changes are on the pictures. Can you please help me ?

**Comment by mckenziearts** [comment]
Hi which version did you use currently?

**Comment by donchoborisov** [comment]
not sure i just follow the instructions. everything else works fine. I think is the last verison 2.0

**Comment by larsvg** [comment]
Same problem on my end. I updated to v2.0 because v1.* broke the laravel 8 update. But now Tailwind takes over my Bootstrap application. h1 tags, links etc. are all normalized.

Is it possible to have a laravel 8 compatible version of the v1 branch?

**Comment by donchoborisov** [comment]
I manage to solve the problem with custom css directly in the elements but that shouldn't be the case...

**Comment by ramonpego** [comment]
the same happened here, i had to make changes in almost all my projects.
Sometimes my project's css break the notification, sometimes the notification css break my site style

**Comment by kgp43** [comment]
Same here

Using a basic Bootstrap4 template and it changes my h1 tags (removes the styling completely).

I can use the Dark Theme, but it will not position bottom-right and it does not auto-close (i need to manually close by clicking the cross).

Using latest version (as of today) - followed the installation guide.

**Comment by donchoborisov** [comment]
Is anyone come up with any solution ?

**Comment by matthbon** [comment]
Is there already a solution for this? I also have the same problem that is changes my own styles

**Comment by Spillo17** [comment]
+1 same here! still no solution?

**Comment by bigint** [comment]
Waiting for the solution 🙇🏻

**Comment by muhsinzyne** [comment]
any solution. i am using laravel 7 with ubold theme purchased

**Comment by colinbarstow** [comment]
I am also experiencing the same issues, has anyone found a work around that doesn't involve completely modifying my existing css files?

**Comment by VinceGee** [comment]
Waiting for solution...which CSS files are you modifying for it to work?

**Comment by mwamodo** [comment]
I think this affects bootstrap. If this happens with tailwind you could just remove the notifyCss directive and it would still work. A solution would be to prefix notifycss class names.

**Comment by zainiafzan** [comment]
Hi have this issues too.. I use tailwind.
after I remove line 3 on notify.css in vendor publish
it works..

**Comment by MarouanAfkir** [comment]
I think putting the @notifyCss before any other css links would solve your problem

**Comment by haexperts2020** [comment]
I am surprised this has gone on so long. Well here is my solution:

https://jsfiddle.net/qct52djv/

Basically took the stock css file and remove everything related to body, html, etc classes. Only thing I kept was the button classes for the "x" on the notification. Hope this helps someone as this was frustrating for me also. 

Just remove @notifyCss and add your custom css style sheet.

**Comment by boriitoo** [comment]
> I am surprised this has gone on so long. Well here is my solution:
> 
> https://jsfiddle.net/qct52djv/
> 
> Basically took the stock css file and remove everything related to body, html, etc classes. Only thing I kept was the button classes for the "x" on the notification. Hope this helps someone as this was frustrating for me also.
> 
> Just remove @notifyCss and add your custom css style sheet.

Thank you very much. This is a working solution for me.

**Comment by MarJose123** [comment]
any solution? or workaround? I have the same issues but for me, it goes behind my form or navbar.

I'm using bootstrap 4 with argon.

**Comment by p1xel007** [comment]
The issue is because this package used "tailwindcss" for some or most of the css styles, and the styles are overriding the other css styles. 

You can just add ".notify" before all the css classes that's conflicting with your styles in "vendor\mckenziearts\laravel-notify\css\notify.css"

**Comment by kgnfth** [comment]
still no fix
Time to run `npm remove laravel-notify`

**Comment by haexperts2020** [comment]
Just remove the stock CSS and copy and paste this one:

https://github.com/mckenziearts/laravel-notify/issues/38#issuecomment-929398051

**Comment by MarJose123** [comment]
I created a similar repo and did a refactor to use Boostrap.

https://github.com/whoami213/laravel-bootstrap-notify

**Comment by JoJo-Bear** [comment]
Our project was affected by this too. It appeared to be the Tailwinds Preflight option. I've submitted this PR #93 which should prevent the styling issues

**Comment by mbia1378** [comment]
Add this in your custom styles file 

.inset-0 {
    z-index: 9999;
    margin-left: 950px;
    margin-top: 65px;
  }
  
  @media only screen and (max-width: 600px) {
    .inset-0 {
        z-index: 9999;
    }
  }

**Comment by damms005** [comment]
I created [a fork that fixes collision with Alpine JS and all other possible style issues](https://github.com/damms005/laravel-notify).

I also updated the readme to make it easier to use this package, so that you won't have Javascript/CSS conflicts anymore.

For now, you simply use the fork by updating the `repositories` section of your `composer.json` file like so (or add it):

```
{
   "repositories": {
       ...
        "laravel-notify": {
            "type": "vcs",
            "url": "https://github.com/damms005/laravel-notify"
        }
    }
}
```

Hopefully, @mckenziearts will approve that I submit it as PR or effect my changes into this original package. Else, if people keep having these issues I may just make the new fork into its own new separate package and continue to try and maintain it and make friendlier to use, because IMHO, this package is too awesome to have such a poor DX.

The authors have done an outstanding job and kudos to them!
